Babylon/groups/api/groups/workspace/commands/update.py

```diff
@@ -0,0 +1,88 @@
+from logging import getLogger
+from pprint import pformat
+from typing import Optional
+
+from click import argument
+from click import command
+from click import make_pass_decorator
+from click import option
+from cosmotech_api.api.workspace_api import WorkspaceApi
+from cosmotech_api.exceptions import NotFoundException
+from cosmotech_api.exceptions import UnauthorizedException
+
+from ......utils.api import get_api_file
+from ......utils.decorators import allow_dry_run
+from ......utils.decorators import require_deployment_key
+from ......utils.decorators import timing_decorator
+
+logger = getLogger("Babylon")
+
+pass_workspace_api = make_pass_decorator(WorkspaceApi)
+
+
+@command()
+@allow_dry_run
+@timing_decorator
+@pass_workspace_api
+@argument("workspace_file")
+@require_deployment_key("sendScenarioMetadataToEventHub", "send_scenario_metadata_to_event_hub")
+@require_deployment_key("useDedicatedEventHubNamespace", "use_dedicated_event_hub_namespace")
+@require_deployment_key("organization_id", "organization_id")
+@require_deployment_key("solution_id", "solution_id")
```

Comment:
Some yaml keys use snake_case, the other use camelCase, it should be uniformized